### ISSUE TITLE
get-client-lib fails with "Unexpected character"

### ISSUE THREAD

**Issue by sparhomenko** [cloudendpoints/endpoints-java#30]
We are using endpoints tool to generate a Java client library for our API:
```bash
java -cp <endpoints-framework-tools and all dependency JARs> \
    com.google.api.server.spi.tools.EndpointsTool \
    get-client-lib \
    --war=<directory of our WAR> \
    <our API class>
```

With `2.0.0-beta.9` everything worked fine:
>API client library written to ./<api name>-v1-java.zip

With `2.0.0-beta.10` it no longer works:

>API Discovery Document written to ./\<api name\>-v1-rest.discovery
>Error: Unexpected character ('.' (code 46)): expected a valid value (number, String, array, object, 'true', 'false' or 'null')
> at [Source: ./\<api name\>-v1-rest.discovery; line: 1, column: 2]

The discovery file is there contains a valid JSON structure. Seeing a similar error message in a Jackson [issue](https://github.com/FasterXML/jackson-core/issues/264) it seems this happens when `ObjectMapper.readTree(String)` is called providing a String with a path to the JSON file instead of its contents.

**Comment by tangiel** [comment]
Thanks for the report. Can you try this as a workaround?

```bash
java -cp <...> \
    com.google.api.server.spi.tools.EndpointsTool \
    get-discovery-doc \
    --war=<...> \
    <...>
java -cp <...> \
    com.google.api.server.spi.tools.EndpointsTool \
    gen-client-lib \
    <discovery doc>
```

**Comment by sparhomenko** [comment]
@tangiel That works, thanks, we'll use that as a workaround for the moment.

**Comment by tangiel** [comment]
This should be fixed by #34 and is being released in beta.12. Thank you for the report. Please let us know if there are still issues.

**Comment by sparhomenko** [comment]
The fix looks good, thanks for your help.
